src/pages/Account/AccountGeneral.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from 'react';
+import { Col, FormGroup, Label, FormFeedback, Row, Button } from 'reactstrap';
+import { compose, graphql } from 'react-apollo';
+import { UPDATE_USER, User, GET_ACCOUNT_PAGE, UserField } from 'networking/users';
+import { Formik, Form, FormikProps, Field, FormikActions } from 'formik';
+import * as Yup from 'yup';
+import Textarea from 'components/shared/Textarea';
+import { TextareaEvent } from 'components/shared/Textarea/Textarea';
+
+interface FormValues {
+  [name: string]: boolean | string | string[] | number | object | undefined;
+}
+
+const GeneralInfoSchema = Yup.object().shape({
+  [UserField.FIRST_NAME]: Yup.string().min(5),
+  [UserField.LAST_NAME]: Yup.string().min(1),
+  [UserField.EMAIL]: Yup.string().min(1),
+  [UserField.ABOUT]: Yup.string().min(1),
+})
+
+const defaultValues: FormValues = {
+  [UserField.FIRST_NAME]: '',
+  [UserField.LAST_NAME]: '',
+  [UserField.EMAIL]: '',
+  [UserField.ABOUT]: '',
+}
+
+function AccountGeneral({ user, updateUser }: any) {
+  return (
+    <Formik
+      initialValues={populateForm(defaultValues, user)}
+      isInitialValid
+      validationSchema={GeneralInfoSchema}
+      onSubmit={handleSubmit}>
+      {({ errors, resetForm, isSubmitting, setFieldValue, values }: FormikProps<any>) => (
+        <Form>
+          <Row>
+            <Col md={6}>
+              <FormGroup inline>
+                <Label for={UserField.FIRST_NAME} className="form-label">First Name</Label>
+                <Field
+                  className={`form-control${errors.firstName ? ' is-invalid' : ''}`}
+                  id={UserField.FIRST_NAME}
+                  name={UserField.FIRST_NAME}
+                  placeholder="First name"
+                  type="text" />
+                <FormFeedback>{errors.firstName}</FormFeedback>
+              </FormGroup>
+            </Col>
+            <Col md={6}>
+              <FormGroup inline>
+                <Label for={UserField.LAST_NAME} className="form-label">Last Name</Label>
+                <Field
+                  className={`form-control${errors.lastName ? ' is-invalid' : ''}`}
```

Comment:
we can use reactstrap `Input` and get pretty colors and stylings

Reply:
swapped